example/__tests__/videoroomContext.test.ts

```diff
@@ -0,0 +1,129 @@
+import { renderHook, act } from '@testing-library/react-hooks';
+
+import {
+  VideoroomContextProvider,
+  useVideoroomState,
+} from '../src/VideoroomContext';
+
+const NOOP = () => {};
+
+jest.mock('@sentry/react-native');
+jest.mock('../src/model/NotificationsContext', () => {
+  return {
+    useNotifications: () => {
+      return { showNotification: NOOP };
+    },
+  };
+});
+jest.mock('../../src/index');
+
+const useCameraStateMock = jest.fn(() => {});
+const useMicrophoneStateMock = jest.fn(() => {});
+const useVideoTrackMetadataMock = jest.fn(() => {});
+const useAudioTrackMetadataMock = jest.fn(() => {});
+
+const membraneWebRTC = require('../../src/index');
+membraneWebRTC.useScreencast = () => {
+  return { isScreencastOn: false, toggleScreencast: () => {} };
+};
+membraneWebRTC.useMembraneServer = () => {
+  return {
+    connect: async (): Promise<void> => {
+      return new Promise<void>((resolve) => {
+        resolve();
```

Comment:
can you put this in the mocks directory? also it's the same as joinRoom, so maybe de-duplicate this

Reply:
Done.